src/apps/backend/modules/share-tasks/internal/share-task-util.ts

```diff
@@ -0,0 +1,15 @@
+//backend\modules\share-tasks\internal\share-task-util.ts
+import { SharedTask } from '../types';
+import { SharedTaskDB } from './store/share-task-db';
+
+export default class SharedTaskUtil {
+  public static convertSharedTaskDBToSharedTask(
+    sharedTaskDb: SharedTaskDB,
+  ): SharedTask {
+    const sharedTask = new SharedTask();
+    sharedTask.id = sharedTaskDb._id.toString();
+    sharedTask.task = sharedTaskDb.task.toString();
```

Comment:
This will not work when task is a object and same goes for account

Reply:
If it is an ObjectId then we can do toString() else we need to create a separate method to convert populated task db to task